community/maintainership.md

```diff
@@ -0,0 +1,102 @@
+<!--
+  SPDX-License-Identifier: CC-BY-4.0
+
+  This file was derived from:
+    https://github.com/hyperledger/toc/blob/gh-pages/guidelines/SAMPLE-MAINTAINERS.md
+-->
+
+# Maintainership
+
+## Active Maintainers
+
+| Name | GitHub | Repo |
+| --- | --- | --- |
+| Andi Gunderson | agunde406 | All |
+| Cian Montgomery | cianx | All |
+| Dan Anderson | danintel | Docs |
+| Dan Middleton | dcmiddle | All |
+| Isabel Tomb | isabeltomb | Lib |
+| James Mitchell | jsmitchell | All |
+| Kelly Olson | ineffectualproperty | Docs |
+| Peter Schwarz | peterschwarz | All |
+| Richard Berg | rberg2 | All |
+| Ryan Banks | RyanLassigBanks | Sabre |
+| Ryan Beck-Buysse | rbuysse | All |
+| Shawn Amundson | vaporos | All |
+| Shannyn Telander | shannynalayna | Swift |
+| Tom Barnes | TomBarnes | Docs |
+
+For a complete list of active and emeritus maintainers, see
+[MAINTAINERS.md](https://github.com/hyperledger/sawtooth-core/blob/main/MAINTAINERS.md).
+
+## Becoming a Maintainer
+
+This community welcomes contributions. Interested contributors are encouraged
+to progress to become maintainers. A contributor can qualify to become
+a maintainer by demonstrating both quality code contributions over a duration
+of time and an understanding (and acceptance) of the project's established
+contribution rules (such as the commit process) and other project and
+Hyperledger norms (such as Discord etiquette).
+
+As a general guideline, maintainers should require a minimum of 6 months of
+contributions and at minimum of 5 non-trivial, high-quality PRs.
+
+To become a maintainer the following steps occur, roughly in order.
+
+- A PR is created by an existing maintainer to update MAINTAINERS.md to add the
+  proposed maintainer to the list of active maintainers.
+- The proposed maintainer confirms their interest in being a maintainer by
+  adding a comment to the PR. The comment from the proposed maintainer must
+  include their willingness
+  to be a long-term (more than 6 month) maintainer.
+- Once the PR and necessary comments have been received, an approval timeframe
+  begins.
+- The PR MUST be communicated on all relevant community calls and the
+  #sawtooth-contributor Discord channel. Comments of support from the community
+  are welcome.
+- The PR is merged and the proposed maintainer becomes a maintainer if either:
+  - Two weeks have passed since at least an absolute majority of maintainer PR
+    approvals have been recorded and no maintainer has dissented, OR
```

Comment:
Absolute majority is tricky, can this be something like "at least 2/3rd of whoever participated from the current maintainers group" in the PR review process?

Reply:
This does intend to say an absolute majority of the maintainers, for this particular type of PR.